core/src/main/scala/kafka/coordinator/GroupCoordinator.scala

```diff
@@ -454,23 +454,45 @@ class GroupCoordinator(val brokerId: Int,
     delayedOffsetStore.foreach(groupManager.store)
   }
 
+  def getPartitions(groupId: String): List[TopicPartition] = {
+      val group = groupManager.getGroup(groupId)
+      group.fold(List[TopicPartition]())(_.allOffsets.keySet.toList)
+  }
 
   def handleFetchOffsets(groupId: String,
-                         partitions: Seq[TopicPartition]): Map[TopicPartition, OffsetFetchResponse.PartitionData] = {
+                         partitions: Seq[TopicPartition],
+                         apiVersion: Short): (Map[TopicPartition, OffsetFetchResponse.PartitionData], Short) = {
+    require(apiVersion >= 1)
+
     if (!isActive.get) {
-      partitions.map { topicPartition =>
-        (topicPartition, new OffsetFetchResponse.PartitionData(OffsetFetchResponse.INVALID_OFFSET, "", Errors.GROUP_COORDINATOR_NOT_AVAILABLE.code))}.toMap
+      (apiVersion match {
```

Comment:
True. I'm not sure that's better or worse. It doesn't seem too bad given that we already return `OffsetFetchResponse.PartitionData` though.